### DISCUSSION TITLE
Roadmap, UI - AutoUpdate like wordpress

### DISCUSSION THREAD

**Discussion by fasteddys** [RaythaHQ/raytha#151] [General]
Hello can we update raytha from inside the UI

Can you share a roadmap of features planned.

If a developer wants to contribute, do we build plugins or new objects into the code?

**Comment by apexdodge** [comment]
Thanks for your question - going to move this over to Discussions and respond there.

**Comment by apexdodge** [comment]
The way to update your deployed raytha platform is one of two ways:

- Publish via visual studio
- Deploy the docker container

Wordpress is built on PHP and Raytha is built on .NET. I'm not aware of a good mechanism in a .NET web application that updates its core files in-place. Raytha is also optimized to run in hyperscaler environment like Azure App Services and Azure Kubernetes Service so that would be a factor as well.

We are currently in the midst of building out the raytha.com Hosted Platform where anybody can spin up a 14 day free trial of Raytha and experiment with the platform. This will expose many more people to Raytha who do not have the time or skillset to run it locally on their machine. It will also have the ability to pay to keep it running beyond 14 days for those who do not wish to host raytha themselves. Goal is to have this released by the end of this year. Very exciting moment for us. After that is released, we will formally decide on the next feature set and roadmap.

Developers that want to contribute - currently it is core objects and functionality, not as plugins.

Thanks!
